pass/disk.go

```diff
@@ -55,7 +58,13 @@ func (s *diskStore) Search(query string) ([]string, error) {
 		return nil, err
 	}
 
+	matches3, err := zglob.GlobFollowSymlinks(s.path + "/**/" + query + "/*/*.gpg")
```

Comment:
I think you lack `*` after the query, so it should be `"/**/" + query + "*/*/*.gpg"` just like in the first example.

Also, I think since you add this, the first case (`DOMAIN/USERNAME.gpg`) just becomes a special case of this one, so we can remove it.

And finally, please add a unit test (in `pass/disk_test.go`) to show that the code does indeed look into subdirectories now.